pkg/workflow_template.go

```diff
@@ -178,6 +178,29 @@ func (c *Client) getWorkflowTemplate(namespace, uid string, version int32) (work
 	return
 }
 
+func (c *Client) getWorkflowTemplateByName(namespace, name string, version int32) (workflowTemplate *WorkflowTemplate, err error) {
```

Comment:
Why do you need a version if you're getting a template by name?

Reply:
@aleksandrmelnikov

Reply:
@Vafilor This chunk of code:
```
	sb := c.workflowTemplatesSelectBuilder(namespace).Where(sq.Eq{"wt.name": name}).
		Columns("wtv.manifest").
		OrderBy("wtv.version desc").
		Limit(1)
	if version != 0 {
		sb = sb.Where(sq.Eq{"wtv.version": version})
	}
```
The SelectBuilder does a join with the workflow_template_versions table.
So there are multiple rows available for a given workflow_template by name, right?
So you can be more explicit with by using version.

Is this line of reasoning correct?

Reply:
Ah, yeah, that makes sense.

Maybe we should add a method later that doesn't take a version or something, but all good.